### PR TITLE
feat: update button styling and align props with material button

### PR DESCRIPTION
**Pull request by twebber90**
### Feature
Update styling of button
Remove `title` prop and instead pass children to button component

- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the documentation
- [X] Any dependent changes have been merged and published in downstream modules
- [X] The test workflow is passing locally

## Screenshots/GIFs
Screen Shot 2021-03-24 at 2 13 20 PM

### CODE REVIEW

File: packages/button/src/styles.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ const loadingSpinner = {
 };
 
 const useStyles = makeStyles((theme) => ({
+    button: {
+        borderRadius: 8,
+        padding: '9px 24px',
+    },
```
Comment (cody-hoffman) [inline packages/button/src/styles.js:12]: ❓ assuming you still need to add the css to make sure the text is always title case?

**Review by cody-hoffman — APPROVED** [review]
❓ were we wanting to release this as a breaking change or are we good with the current sites getting the changes?

File: packages/button/src/styles.js (hunk 1 above)
Comment (twebber90) [reply packages/button/src/styles.js:12]: correct. Been in a meeting since getting that feedback from Jeff

**Comment by twebber90** [comment]
@cody-hoffman I ran the commit wizard to do a breaking change but I don't see it anywhere... it is definitely breaking

**Pull request merged**